ADD: Documentation for Inventory Module and Closeable Menu (#180)
ink-inc/MagicMittelalterGame: doc/developer_guide/Menu/InventoryContextMenu.adoc

=== Responsiblity

> This Menu inheritates from link:CloseableMenu.adoc[CloseableMenu]. The `Show()` method is overriden to include

Reviewer: ```suggestion
This Menu inherits from link:CloseableMenu.adoc[CloseableMenu]. The `Show()` method is overridden to include
```

> Instantiation of the needed buttons. The primary context action button (`Consume`, `Use`, `Equip`) is added here.

Reviewer: ```suggestion
instantiation of the needed buttons. The primary context action button (`Consume`, `Use`, `Equip`) is added here.
```

If the link:../Inventory/InventoryItem.adoc[InventoryItem] can be dropped, a `Drop`-Button is also shown.

> `Hide()` is overridden to Destroy the created context action buttons.

Reviewer: ```suggestion
`Hide()` is overridden to destroy the created context action buttons.
```
